tests/io/test_ase.py

```diff
@@ -292,10 +297,14 @@ def test_back_forth(self):
         # Molecule --> Atoms --> Molecule --> Atoms
         molecule = Molecule.from_file(TEST_FILES_DIR / "acetylene.xyz")
         molecule.set_charge_and_spin(-2, spin_multiplicity=3)
-        molecule.info = {"test": "hi"}
+        molecule.properties = {"test": "hi"}
         atoms = aio.AseAtomsAdaptor.get_atoms(molecule)
         molecule_back = aio.AseAtomsAdaptor.get_molecule(atoms)
         atoms_back = aio.AseAtomsAdaptor.get_atoms(molecule_back)
         for k, v in atoms.todict().items():
             assert str(atoms_back.todict()[k]) == str(v)
         assert molecule_back == molecule
```

Comment:
This doesn't currently check that the `properties` were restored since `Molecule.__eq__` doesn't currently look at `properties`. I think that'll be unexpected behavior for must users so we should change `IMolecule.__eq__` and `IStructure.__eq__` even though it's strictly speaking a breaking change.